test/fixture/types.ts

```diff
@@ -15,90 +15,94 @@ describe('API routes', () => {
       | string | number | TestResponse
       | { testFile: string, hasEnv: boolean }
       | { internalApiKey: string }
-      | { [key: string]: any }
-      >>() // middleware
-    expectTypeOf($fetch('/api/unknown')).toMatchTypeOf<Promise<unknown>>()
-    expectTypeOf($fetch<TestResponse>('/test')).toMatchTypeOf<Promise<TestResponse>>()
+      | {
+        depA: any
+        depB: any
+        depLib: any
```

Comment:
Can we somehow include only some routes? I find it too hard to update type tests each time by adding a route to the fixture.

Reply:
update to use `InternalApi` to infer all possible return types instead of manually defining each time in #590